src/app/dbtest/page.tsx

```diff
@@ -0,0 +1,40 @@
+import { connectMongo } from "@/lib/db";
+import UserModel, { UserRole, MeetingPreference } from "@/lib/User/model";
+import React from "react";
+
+const dbTest = () => {
+   connectMongo();
```

Comment:
This is an async function and must be `await`ed or `.then()`

Reply:
converted to async